src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/AddDefendantLitigationFriendCallbackHandler.java

```diff
@@ -47,6 +59,7 @@
         return EVENTS;
     }
 
+    //production uses this callback until CCD change is merged
```

Comment:
When both of these changes get merged, will you create a PR to remove this old production callback. As it will be pointing to V_1 & the generic field will be live

Reply:
@simms11 advised me earlier that the old callback will be removed. Probably better to raise a tech backlog ticket for cleanup task